### PR TITLE
feat: Add error banner and loading indicator

### LINKED ISSUE #230: feat(web): Show some info on failed calculations

### PR DESCRIPTION
**Pull request by ingeridhellen**
Refactor into a status component

## Why is this pull request needed?

This pull request is needed because users should get a warning if things fail

## What does this pull request change?

- Adds an error banner if failing
- Adds a process indicator when calculating
- Change default message into an info banner
- Refactor into a status component

## Issues related to this change:

Closes #230

### CODE REVIEW

**Review by rikkebl — APPROVED** [review]
Looks good!

File: web/src/feature/Status.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return <></>
+      }
+    case 'failure':
+      return (
+        <EdsBanner>
+          <EdsBanner.Icon variant="warning">
+            <Icon data={error_outlined} />
+          </EdsBanner.Icon>
+          <EdsBanner.Message>
+            An unexpected error has occurred. Please inform us if it persists
```
Comment (soofstad) [inline web/src/feature/Status.tsx:38]:
```suggestion
            An unexpected error has occurred. Please inform the system responsible if the problem persists
```

**Review by soofstad — APPROVED** [review]
Good refactor!

File: web/src/feature/Status.tsx (hunk 1 above)
Comment (ingeridhellen) [reply web/src/feature/Status.tsx:38]: I agree, that was better. Already merged it though, but may fix it later

**Pull request merged**